Core/DISET/private/BaseClient.py

```diff
@@ -47,6 +47,9 @@ def __init__( self, serviceName, **kwargs ):
     self._destinationSrv = serviceName
     self._serviceName = serviceName
     self.kwargs = kwargs
+    self.useCertificates = None
```

Comment:
Can we preppend one or two underscore to these variable names to make sure they are not set from outisde ?